packages/studio-plugin/src/parsers/StudioSourceFileParser.ts

```diff
@@ -123,19 +124,29 @@ export default class StudioSourceFileParser {
   parseCssImports(): string[] {
     const cssImports: string[] = [];
 
-    const resolveRelativeFilepath = (filepath: string) => {
-      if (filepath.startsWith(".")) {
-        return upath.resolve(
-          upath.dirname(this.sourceFile.getFilePath()),
-          filepath
+    const getImportAbsoluteFilepath = (importPath: string) => {
```

Comment:
what do you think of `getAbsoluteImportFilepath` instead?

Reply:
I like it!